test/Swashbuckle.AspNetCore.Newtonsoft.Test/SchemaGenerator/NewtonsoftSchemaGeneratorTests.cs

```diff
@@ -161,6 +161,30 @@ public void GenerateSchema_GeneratesReferencedDictionarySchema_IfDictionaryTypeI
             Assert.Equal(expectedItemsFormat, schema.Items.Format);
         }
 
+        [Fact]
+        public void GenerateSchema_GeneratesBaseType_IfGenericInHierarchy()
+        {
+            var subject = Subject(
+                configureGenerator: c => c.UseAllOfForInheritance = true
+            );
+
+            var schemaRepository = new SchemaRepository();
+
+            var referenceSchema = subject.GenerateSchema(typeof(SubTypeOfGeneric), schemaRepository);
```

Comment:
Can we also assert on the names of the properties that are found? Or is that not worth doing?